content/en/docs/concepts/configuration/secret.md

```diff
@@ -854,6 +854,43 @@ start until all the Pod's volumes are mounted.
 
 ## Use cases
 
+### Use-Case: As container environment variables
+
+Create a secret
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysecret
+type: Opaque
+data:
+  username: YWRtaW4=
+  password: MWYyZDFlMmU2N2Rm
+```
+
+Create the Secret:
+```shell
+kubectl apply -f mysecret.yaml
+```
+
+Use `envFrom` to define all of the Secret’s data as container environment variables. The key from the Secret becomes the environment variable name in the Pod.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dapi-test-pod
+spec:
+  containers:
+    - name: test-container
+      image: k8s.gcr.io/busybox
```

Comment:
The other examples use fictional container images such as `myClientImage`. I recommend that this addition does the same.

Reply:
em ... using a working container may be a better choice. Most users would try copy/paste and run the examples. So ... maybe instead we should change all other examples, in a follow up PR?

Reply:
As I copy it from the configMap example.
https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables

I agree that the document should remain highly consistent in order to let the new joiner easily to join a powerful k8s family.

What do u think @sftim  and @tengqm